src/CodeTracking.jl

```diff
@@ -252,6 +256,29 @@ end
 
 definition(method::Method) = definition(Expr, method)
 
+"""
+    code_expr(f, types)
+
+Returns the expression for the method definition for `f` with the specified types.
+
+May return `nothing` if Revise isn't loaded. In such cases, calling
+`Meta.parse(code_string(f, types))` can sometimes be an alternative.
+"""
+code_expr(f, t) = definition(Expr, which(f, t))
+macro code_expr(ex0...)
+    InteractiveUtils.gen_call_with_extracted_types_and_kwargs(__module__, :code_expr, ex0)
```

Comment:
there is a builtin function for this?!
This is dangerously close to something I need in ExprTools